docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- This option is not highly recommended for new users as it is more complicated and slightly less flexible, but is included here since it allows use of 900 MHz ELRS receivers you may already have.
+- This Tx module uses the SX1276 LoRa chip and supports only 19 Hz mode.  It works with many 900 MHz ELRS receivers but is incompatible with the MatekSys gear above.
+- The FrSky R9M can be flashed using your EdgeTX or OpenTX radio.
+- No 3D printer is required, but transmit power should be limited to less than 500 mW unless you add a fan.
+- The Bayck 915M Nano Pro ELRS receiver can be [flashed](ELRS_RECEIVERS.md#flashing) with mLRS firmware without removing it from your vehicle by using ArduPilot serial passthrough.
+- If you already have a R9MX or R9MM receiver, they are also compatible, but have lower power and are more difficult to [flash](FRSKY_R9.md#flash-r9-receivers-with-elrs-bootloader).
+- Unfortunately, the R9M has inverted serial port signals; if you choose this option you will probably also want to purchase an [ESP32 module](FRSKY_R9.md#esp32-wireless-bridge) such as the [M5 Stamp Pico Mate](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) to use as a wireless bridge and handle the inverted serial port.
+
+## 2. Assemble your Tx module if required and install antennas on both the receiver and Tx module
+
+If you are using one of the MatekSys Tx module kits, print or purchase a JR bay case from the STL download available from [MatekSys](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-1).
+
+If you will use the MatekSys HC04 module to connect your ground station via Bluetooth, set all three [dip switches](https://MATEKSYS.md#tx-module-hc-04-bluetooth-notes) to the on position before assembling your Tx module.
+
+If you are using the [R9M Tx module](FRSKY_R9.md#r9m-tx-module):
+- Set dip switch 1 to the On position to configure it for serial data
+- Install the [Arduino IDE](https://www.arduino.cc/en/software) and use it to build and flash [firmware](WIRELESS_BRIDGE.md) on your ESP32 module.  Be sure to select the R9M variant for your module when you edit the mlrs-wireless-bridge.ino.
+- Connect your [ESP32 wireless bridge](https://FRSKY_R9.md#esp32-wireless-bridge) module to the serial and power pins of your R9M.
+
+When choosing and connecting antennas be aware of the difference between SMA and RP-SMA.  It is, unfortunately, possible to attach an RP-SMA antenna to an SMA jack but there will be no electrical connection of the signal.
+
+## 3. Connect your receiver to your flight controller
+
+For the 1 watt MatekSys [mLRS receivers](https://www.mateksys.com/?portfolio=mr24-30#tab-id-2):
+- connect the receiver Rx1 to a FC UART Tx.
+- connect the receiver Tx1 to the corresponding FC UART Rx.
+- Connect the receiver V pin to a 5 volt output on your FC.
+- Connect the receiver G pin to a ground pad on your FC.
+- If you will use CRSF for RC channels (recommended), connect the receiver TX2 to a different UART Rx on your FC.
```

Comment:
is it correct that CRSF is the recommended method?

Reply:
You tell me.  I almost never use CRSF because I have very few receivers which support it and I also have some FC with too few serial ports.  I fixed a problem with sending link quality on a channel. But at that time, you didn't seem to think the fix was very important, I assumed because CRSF was the main use case.  As far as I know, CRSF is still the only way to get full functionality in terms of radio link info with released versions of ardupilot.  So, shouldn't we recommend it be connected when it is available?

Reply:
it's unfortunately true that ArduPilot is doing the DJI in this regards, but I notice too that a good number user it without that wire. I think just kill the bracket and we are on the neutral = safe side :)

Reply:
I don't think we need to be neutral.  Recommending things is part of what I think a getting started guide should do.
ELRS is very popular so some new users may have used it.  I am guessing that ELRS get all the link stat goodness CRSF has to offer so I was thinking it safer to recommend it here to avoid disappointment and questions about why some link statistics are missing.

Reply:
we sure don't need to but we can when we find appropriate
ELRS mavlink-rc users won't get a single dime more stats than we ...

I just would remove the "(recommeneded)" and all good

Reply:
OK.